docs/GenericsManifesto.md

```diff
@@ -181,6 +186,8 @@ There are a number of minor extensions we can make to the generics system that d
 
 ### Arbitrary requirements in protocols (*)
 
+*This feature has been accepted in [SE-0142](https://github.com/apple/swift-evolution/blob/master/proposals/0142-associated-types-constraints.md) and is planned for release in SWIFT_VERSION_HERE.*
```

Comment:
Can someone from the core team confirm the version of Swift this will be released in? 3.1 or 4?